README.md

```diff
@@ -2,69 +2,71 @@
-                   │  converter API   │
-                   └─────────┬────────┘
-                             │
-                             │
-                             │
-                       ┌─────▼─────┐
-                       │ Saltstack │
-                       └─────┬─────┘
-                             │
-                             │
-                             │
-                             │
-                    ┌────────▼────────┐
-                    │ Network devices │
-                    └─────────────────┘
+Our approach to automation is opinionated. There are tons of ways of doing network configuration, and choices must be made.
+
+                       ┌──────────────┐     ┌────────────────────┐
+                       │ Network CMDB │     │ Other data sources │
+                       └──────────┬───┘     └──┬─────────────────┘
+                                  │            │
+                                  │            │
+                                  │            │
+                                  │            │
+                               ┌──▼────────────▼──┐
+                               │ Data aggregation │
+                               │        API       │
+                               └─────────┬────────┘
+                                         │
+                                         │
+                                         │
+                                   ┌─────▼─────┐
+                                   │ SaltStack │
+                                   └─────┬─────┘
+                                         │
+                                         │
+                                         │
+                                         │
+                                ┌────────▼────────┐
+                                │ Network devices │
+                                └─────────────────┘
 
 ### Network CMDB
 
-The Network CMDB contains data relative to the business and is completely agnostic to the network OS. The models are designed to describe the objects themselves rather than the configuration from devices perspective. The idea is also to avoid any data duplication which could lead to mismatched configuration.
+The Network CMDB contains data relative to the business and is completely agnostic to the network OS.
 
-For instance, we represent the BGP session itself with two joined table describing peers: DeviceBGPSession <==> BGPSession <==> DeviceBGPSession.
-* DeviceBGPSession contains the local-as but not the peer-as, avoiding data duplication. The peer-as being the local-as of the other neighbor.
-* BGPSession contains all information peers have in common, like status (in production, maintenance etc...) or MD5 password
+The models are designed to describe the objects themselves rather than the configuration from device perspective. The idea is also to avoid any data duplication which could lead to configuration mismatchs.
```

Comment:
The models  follow a network centric approach, not a device centric approach. This prevents duplication of information across multiple devices.